src/api/getDataFromBackend.ts

```diff
@@ -3,7 +3,7 @@ import { GraphQLClient, RequestDocument } from 'graphql-request';
 const getDataFromBackend = async ({ query, variables, preview, }: { query: RequestDocument, variables?: unknown, preview?: boolean, }) => {
   const endpoint = 'https://graphql.datocms.com/';
 
-  const includeDrafts = preview ?? process.env.CONTEXT !== 'production';
```

Comment:
Can't we usue the Netlify env variable to check for production, instead of adding a new env variable?

Reply:
Yeah, good point. Have done so here:
https://github.com/RandomStudio/random-studio/pull/181/commits/fbcd6a7e60ba1cd461fe6338c03a1a38dadc91ad

I believe the issue before was that we can't access the Netlify Env directly from Next, which is why it failed. 

Have followed Netlify's advice here:
https://docs.netlify.com/integrations/frameworks/environment-variables/

Why are their docs so confusing these days, everything is buried. Found this via Google.